components/gallery/GalleryItemHead.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div></div>
```

Comment:
How about writing a `useGalleryItemHead` hook or utils function for it, as there is no UI content in this component